python/tests/test_lowlevel.py

```diff
@@ -3,7 +3,10 @@
 import re
 from deltachat import capi
 from deltachat.capi import ffi
-import queue
+try:
+    from Queue import queue
+except ImportError:
+    from queue import Queue
 
```

Comment:
Wat? :)

Reply:
why not also try `from queue import queue` and from `Queue import Queue` :)

Reply:
all because @hpk42 wants to support an ancient version of Python ;)